src/main/scala/beam/utils/beamToVia/IO/Reader.scala

```diff
@@ -0,0 +1,241 @@
+package beam.utils.beamToVia.IO
```

Comment:
Please, do not use upper case in the namespace because it will cause some issue during launching the app run in Linux (case sensitive file system). By JAVA convention [Naming a Package](https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html): Package names are written in all lower case to avoid conflict with the names of classes or interfaces.

Reply:
ok